pegasus/sites.v3/hourofcode.com/views/home_twitter_embed_single_es.html

```diff
@@ -0,0 +1 @@
+<blockquote class="twitter-tweet" data-lang="es" data-dnt="true"><p lang="en" dir="ltr">We want to say THANK YOU to every teacher, student, volunteer, advocate, and parent who participated in <a href="https://twitter.com/hashtag/HourOfCode?src=hash&amp;ref_src=twsrc%5Etfw">#HourOfCode</a>. We&#39;re honored to work w/ you to ensure every student in every classroom has the opportunity to explore, play, and create w/ computer science. See you next year! ❤️ <a href="https://t.co/mtaqgh4ziQ">pic.twitter.com/mtaqgh4ziQ</a></p>&mdash; Code.org (@codeorg) <a href="https://twitter.com/codeorg/status/1602332561438556161?ref_src=twsrc%5Etfw">12 de diciembre de 2022</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
this looks like a mix of spanish and english. should all this text be translated into spanish?

Reply:
It's imported from Twitter so I'm not sure how to translate the text. I tried deleting the `p` text in the code here, but it still showed up in the browser? The surrounding widget text and links are all in Spanish so I thought that was better than nothing. I can talk to Sam about it though if we think this isn't good UX.